src/consts.ts

```diff
@@ -4,3 +4,41 @@
  * _Exported as `paginationSize`_.
  */
 export const paginationSize = 8;
+
+/**
+ * Interface for defining structure of a received POST request
+ */
+export interface ICreateGist {
```

Comment:
Has someone else made these before where we could import them from?

Reply:
I don't know. It's just the structure of the JSON data that we fetch from GitHub Gist so I thought to write my own custom one. 

Anyway, it's not in one of the libraries we already have. Also, downloading another package for this doesn't make sense.